### PR TITLE
Support `log_configuration` param

### PR DESCRIPTION
**Pull request by winebarrel**
Support `log_configuration` param:

```yml
...
app:
  image: busybox
  memory: 128
  log_configuration:
    log_driver: awslogs
    options:
      awslogs-group: my-logs
      awslogs-region: ap-northeast-1
      awslogs-stream-prefix: example
```

### CODE REVIEW

File: lib/hako/schedulers/ecs.rb [hunk 1]
```diff
@@ -438,6 +438,7 @@ def volumes_definition
       # @return [Hash]
       def create_definition(name, container)
         environment = container.env.map { |k, v| { name: k, value: v } }
+        log_configuration = Hash[container.log_configuration.map { |k, v| [k.to_sym, v] }]
```
Comment (eagletmt) [inline lib/hako/schedulers/ecs.rb]: Is `#to_sym` necessary here?

Comment (eagletmt) [inline lib/hako/schedulers/ecs.rb]: Only `log_driver` and `options` should be accepted.

**Comment by eagletmt** [comment]
Put your example usage in examples directory.

File: lib/hako/schedulers/ecs.rb (hunk 1 above)
Comment (winebarrel) [inline lib/hako/schedulers/ecs.rb]: Thanks. fixed it da8c21a

**Comment by winebarrel** [comment]
Add example yaml a3cacda

**Pull request merged**